libs/ui-watt/src/lib/components/toast/watt-toast.service.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Injectable } from '@angular/core';
+import { MatSnackBar, MatSnackBarRef } from '@angular/material/snack-bar';
+
+import { WattToastComponent } from './watt-toast.component';
+import { WattToastConfig } from './watt-toast.component';
+
```

Comment:
nitpick: merge imports in one line. SonarCloud had marked this as "code smell" when there are two separate imports from the same file.

```suggestion
import { WattToastComponent, WattToastConfig } from './watt-toast.component';

```